concepticondata/conceptlists/Desrochers-2010-330.tsv

```diff
@@ -0,0 +1,331 @@
+Desrochers-2010-330-210	210	punta	tip	TIP (OF OBJECT)	2992	Noun	5.89114925588637	5	232	5.69
+Desrochers-2010-330-211	211	puro	pure			Noun	5.42156862745098	4	236	5.33
+Desrochers-2010-330-212	212	rabia	rabies	RABIES	2206	Noun	5.51960784313726	5	128	6.18
+Desrochers-2010-330-213	213	rata	rat	RAT	1490	Noun	5.79411764705882	4	70	6.3
+Desrochers-2010-330-214	214	rayo	ray			Noun	5.15686274509804	4	111	5.86
+Desrochers-2010-330-215	215	sello	stamp	POSTAGE STAMP	596	Noun	6.09915634874148	5	35	6.61
+Desrochers-2010-330-216	216	submarino	submarine			Noun	4.57843137254902	9	12	
+Desrochers-2010-330-217	217	techo	roof	ROOF	769	Noun	5.99019607843137	5	225	6.54
+Desrochers-2010-330-218	218	torpedo	torpedo			Noun	3.90196078431373	7	4	
+Desrochers-2010-330-219	219	uña	nail	FINGERNAIL	1258	Noun	6.23529411764706	3	24	6.27
+Desrochers-2010-330-220	220	uva	grape	GRAPE	920	Noun	5.80392156862745	3	25	
+Desrochers-2010-330-221	221	vajilla	tableware			Noun	5.63725490196079	7	17	
+Desrochers-2010-330-222	222	vampiro	vampire			Noun	4.17647058823529	7	22	4.25
+Desrochers-2010-330-223	223	vaso	glass	GLASS	604	Noun	6.58823529411765	4	209	6.3
+Desrochers-2010-330-224	224	vela	sail			Noun	5.67647058823529	4	83	5.78
+Desrochers-2010-330-225	225	vida	life	LIFE	210	Noun	6.59803921568627	4	4765	6.44
+Desrochers-2010-330-226	226	vino	wine	WINE	1524	Noun	6.35294117647059	4	716	6.39
+Desrochers-2010-330-227	227	abatir	download			Verb	3.77450980392157	6	5	
+Desrochers-2010-330-228	228	abrir	open			Verb	6.55882352941176	5	275	6.45
+Desrochers-2010-330-229	229	acudir	go to			Verb	4.82352941176471	6	141	4.91
+Desrochers-2010-330-230	230	afilar	sharpen	SHARPEN (SOMETHING)	251	Verb	4.63725490196079	6	6	
+Desrochers-2010-330-231	231	aguar	water down			Verb	3.73529411764706	5		
+Desrochers-2010-330-232	232	alumbrar	to illuminate			Verb	4.56862745098039	8	16	4.77
+Desrochers-2010-330-233	233	apaciguar	appease			Verb	3.5	9	6	
+Desrochers-2010-330-234	234	apagar	turn off			Verb	6.5	6	39	5.97
+Desrochers-2010-330-235	235	asir	summarize			Verb	2.43572184416938	4	6	
+Desrochers-2010-330-236	236	atrapar	trap			Verb	4.95098039215686	7	55	5.29
+Desrochers-2010-330-237	237	avisar	notify			Verb	6.02941176470588	6	45	5.81
+Desrochers-2010-330-238	238	balbucear	babble			Verb	3.46078431372549	9	5	
+Desrochers-2010-330-239	239	besar	kiss	KISS	578	Verb	6.52941176470588	5	34	6.85
+Desrochers-2010-330-240	240	brotar	sprout	SPROUT (VERB)	2076	Verb	4.89	6	26	4.13
+Desrochers-2010-330-241	241	bullir	boil	BOIL	2493	Verb	2.41176470588235	6	1	
+Desrochers-2010-330-242	242	buscar	search	LOOK FOR	1468	Verb	6.29411764705882	6	457	6.39
+Desrochers-2010-330-243	243	calar	stall			Verb	4.2843137254902	5	3	
+Desrochers-2010-330-244	244	calcificar	calcify			Verb	3.08823529411765	10		
+Desrochers-2010-330-245	245	calentar	heat	HEAT UP	1341	Verb	6.41176470588235	8	31	6.74
+Desrochers-2010-330-246	246	cantar	sing	SING	1261	Verb	6.34313725490196	6	137	6.21
+Desrochers-2010-330-247	247	cascar	click			Verb	4.65686274509804	6	2	
```

Comment:
crack (a walnut)

Reply:
+1, but in some context might be closer to "to break"

Reply:
I'll leave it unmapped because we do not have a concept for "crack" and it seems to be ambiguous.

Reply:
Might be better. Just one thing: considering there is already a word for "crack/break", maybe it is intended in the secondary meaning, which is actually the etymological one [and cognate with Italian "cassare"]), which is "to hit (repeatedly)"